SIP/sip-45.md

```diff
@@ -376,15 +376,15 @@ While the creator is uploading the contract to the chain, the individual data st
 <br>Id of the token which can be staked on the contract.<br/>
 
 **digitsFactorPledgeToken**<
-<br>Multiplyer for the ratio caluclation by digits of the **pledgetoken**<br/>
-0 digits = 1 , 1 digt = 10 ... 8 digits = 100000000
+<br>Multiplyer for the ratio calculation by digits of the **pledgetoken**<br/>
+0 digits = 1 , 1 digit = 10 ... 8 digits = 100000000
```

Comment:
I still believe there is no need to have any factor inside contracts, this can probably be handled outside. But I need to review all the logic to suggest any changes.